_pages/02-visualizations.md

```diff
@@ -2,8 +2,75 @@
+
+## Packet retransmission vs. download throughput for ISPs in the US
+
+Each bubble corresponds to an ISP. When you mouse over any bar, the chart displays the name of the corresponding ISP.
+The size of any bubble is proportional to the number of tests run in that ISP, while the color of the bubble represents the RTT of that ISP.
+
+<p><iframe class="customIframe" src="https://www.google.com/publicdata/embed?ds=e9krd11m38onf_&amp;ctype=b&amp;strail=false&amp;bcs=d&amp;nselm=s&amp;met_s=number_of_tests&amp;scale_s=lin&amp;ind_s=false&amp;met_y=download_throughput&amp;scale_y=log&amp;ind_y=false&amp;met_x=segretrans_datasegsout&amp;scale_x=log&amp;ind_x=false&amp;met_c=rtt&amp;scale_c=lin&amp;ind_c=false&amp;ifdim=country_isp:country:840&amp;hl=en_US&amp;dl=en_US&amp;ind=false&amp;icfg&amp;iconSize=0.5" name="customIframe_5" width="960" height="500" frameborder="0" marginwidth="960" marginheight="500"></iframe></p>
+
+## Network- versus client-limited tests
+
+Each bubble corresponds to a country. When you mouse over any bubble, the chart displays the name of the corresponding country.  The color of any bubble represents the median download throughput of a country, while the size of the bubble is proportional to the number of tests run in that country.  Bubbles at the top left correspond to countries where throughput is mostly limited by network congestion, while bubbles at the bottom right correspond to countries where throughput is mostly limited by the user's device.
+
+<p><iframe class="customIframe" src="https://www.google.com/publicdata/embed?ds=e9krd11m38onf_&amp;ctype=b&amp;strail=false&amp;bcs=d&amp;nselm=s&amp;met_s=number_of_tests&amp;scale_s=lin&amp;ind_s=false&amp;met_c=download_throughput&amp;scale_c=lin&amp;ind_c=false&amp;met_y=network_limited&amp;scale_y=lin&amp;ind_y=false&amp;met_x=receiver_limited&amp;scale_x=lin&amp;ind_x=false&amp;ifdim=country&amp;hl=en_US&amp;dl=en_US&amp;ind=false&amp;icfg&amp;iconSize=0.5" name="customIframe_6" width="960" height="500" frameborder="0" marginwidth="960" marginheight="500"></iframe></p>
+
+{% endcapture %}
+
+{% capture accordion_entry_2 %}
+
+View a quick and dreamy video showing packet loss, as measured by M-Lab's NDT.
+
+<p><iframe class="customIframe" src="https://docs.google.com/file/d/0B0dPEIVgbGveU1JjcnU3LXNNYXM/preview" name="customIframe_0" width="800" height="400" frameborder="0" marginwidth="800" marginheight="400"></iframe></p>
+
+If you're inspired to learn more, and maybe want to create your own, go to:  [https://dmadev.com/2012/12/10/visualizing-m-lab-data-with-bigquery-part-two/](https://dmadev.com/2012/12/10/visualizing-m-lab-data-with-bigquery-part-two/).
```

Comment:
@shredtechular If you could update this link we should be good to go with this PR.
http://dominichamon.com/blog/2012/12/visualizing-m-lab-data-with-bigquery-part-two/